src/main/java/org/opensearch/knn/index/codec/KNNCodecVersion.java

```diff
@@ -74,9 +77,24 @@ public enum KNNCodecVersion {
             .knnVectorsFormat(new KNN940PerFieldKnnVectorsFormat(Optional.ofNullable(mapperService)))
             .build(),
         KNN940Codec::new
+    ),
+
+    V_9_5_0(
```

Comment:
[Question]: The other enums which are present in the like V_9_4_0, are they getting used somewhere? if not lets remove them.

Reply:
they are referenced from codec classes, like [here](https://github.com/opensearch-project/k-NN/blob/2.x/src/main/java/org/opensearch/knn/index/codec/KNN920Codec/KNN920Codec.java#L23). So if we keep older codecs we also need that other enum members.

Reply:
For clarification - in enum we do have only codec versions starting from 9.1 (9.1.0, 9.2.0, 9.4.0). There are no references for older codecs, so it's not a problem if we decide to drop them in future to be consistent with what core OS supports.